cards/acute-limb-ischemia/card.md

```diff
@@ -34,16 +34,16 @@ categories:
 - **Neurological:** Check for sensory and motor loss
 - **Vascular:** Grade peripheral pulses
 
-- 0 = No Doppler signal
-- 1 = Reduced
-- 2 = Normal
-- 3 = Increased
-- 4 = Bounding
+    - 0 = No Doppler signal
```

Comment:
If you want to make changes, just make the changes in VS Code, on this same branch. Commit your changes. And then use the `sync` button to push your changes. It will update here in the pull request automatically. The pull request follows your branch.